### ISSUE TITLE
Connectivity Tests HTTP Proxy Output

### ISSUE THREAD

**Issue by joaotomazio** [Azure/Azure-Workload-Backup-Troubleshooting-Scripts#15]
When running this script behind a HTTP proxy and with HTTP proxy settings properly configured, I still couldn't succeed on integrating my Azure Windows VM with Azure Backup Service, in order to perform Azure Backup for SQL Server.

As I can understand, the proxy settings need to be configured system-wide, i.e., not only to the logged user but also to the `NT Authority\System` and `NT Service\AzureWLBackupPluginSvc` users, used by the Azure Workload Backup Plugin.

On the `Get-ProxySettings.ps1`, there is a verification on the Windows Registry Key `ProxySettingsPerUser` on `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\CurrentVersion\Internet Settings` in order to check if proxy configurations are assigned system-wide. According to the script, system-wide configurations (meaning, every user gets the same configs) are enabled when this Registry Key either not exists or is assigned with the `0` value.

`Logger "Are Proxy Settings common for all users: $(( $proxyPerUser.GetValue('ProxySettingsPerUser') -eq "0") -or ( $proxyPerUser.GetValue('ProxySettingsPerUser') -eq $null))"`

After a few tests, I've only managed to integrate my VM with Azure Backup by explicitly setting the Registry Key to 0, because it doesn't exist by default.

In a nutshell, my question is if the testing script should only confirm system-wide http proxy configurations with the registry key properly set to 0.

Thanks

**Comment by shashwattrivedi** [comment]
@joaotomazio were you able to start backup for SQL databases after you had set the `ProxySettingsPerUser` to 0?

**Comment by joaotomazio** [comment]
> @joaotomazio were you able to start backup for SQL databases after you had set the `ProxySettingsPerUser` to 0?

Yes
